comie.py

```diff
@@ -24,8 +24,14 @@ def mentionUser(user):
 class Comie(discord.Client):
     ### REACIONS
     async def on_raw_reaction_add(self, payload):
+        #The CS-Dating Channel ID so only 6 thumbs-up will start an event in the CS channel
+        if payload.channel_id == 804047115164712990:
```

Comment:
Replace that for a config entry maybe?

Reply:
Yeah had that on my mind but put that on the later stack:D

Reply:
Add it to the config pls so "config["csgo"]["channel_id"]" should work